ui/app/AppLayouts/Profile/panels/ProfileShowcasePanel.qml

```diff
@@ -30,8 +30,11 @@ DoubleFlickableWithFolding {
     property string emptyInShowcasePlaceholderText
     property string emptyHiddenPlaceholderText
 
-    // Signal to requst position change of the visible items
+    property int showcaseLimit: ProfileUtils.showcaseLimit
```

Comment:
/ui/app/AppLayouts/Profile/panels/ProfileShowcasePanel.qml:33:5: Unable to assign [undefined] to int

Looks like we need to override this property in storybook

Reply:
I cannot see the warning log but I've explicilty added the showcaseLimit for all realted panels.